2020-02-07-roman-to-integer/solution.rb

```diff
@@ -17,9 +17,12 @@ def self.numeral_to_integer(numeral)
       char = chars[index]
       next_char = chars[index + 1]
 
-      if (char == 'I' && %w(V X).include?(next_char)) ||
-         (char == 'X' && %w(L C).include?(next_char)) ||
-         (char == 'C' && %w(D M).include?(next_char))
+      if (char == 'I' && %w(V X L C D M).include?(next_char)) ||
+         (char == 'V' && %w(X L C D M).include?(next_char)) ||
+         (char == 'X' && %w(L C D M).include?(next_char)) ||
+         (char == 'L' && %w(C D M).include?(next_char)) ||
+         (char == 'C' && %w(D M).include?(next_char)) ||
+         (char == 'D' && %w(M).include?(next_char))
```

Comment:
Would it be sufficient to check if the value of `next_char` is greater than `char` instead of all these different cases?

```ruby
if NUMERALS[next_char] && (NUMERALS[next_char] > NUMERALS[char])
  # ...
end
```

Reply:
Wowww I didn't see that way! Thanks a lot for your time!